classes/migration/upgrade/v3_4_0/I8508_ConvertCurrentLogFile.php

```diff
@@ -113,11 +112,6 @@ public function getPhpDateTimeFormat(): string
         return 'Y-m-d H:i:s';
     }
 
-    public function isPathInfoDisabled(): bool
```

Comment:
Because this function is needed to be implemented -- it is an abstract function in ConvertLogFile, maybe to here only return false ?